### PR TITLE
Added option to keep audio when a wired headset is connected

### PR DESCRIPTION
**Pull request by gergesh**
I've added a toggle for not muting videos by default when a wired headset is connected.

### CODE REVIEW

File: Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/ImageViewerController.java [hunk 1]
```diff
@@ -125,7 +127,9 @@ public ImageViewerController(Loadable loadable, Context context, Toolbar toolbar
         this.toolbar = toolbar;
         this.loadable = loadable;
 
-        presenter = new ImageViewerPresenter(this);
+        AudioManager audioManager = (AudioManager) context.getSystemService(AUDIO_SERVICE);
```
Comment (Adamantcheese) [inline Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/ImageViewerController.java]: You can move all this code up into the ImageViewerPresenter constructor itself. No need to modify extra files if it's only used in one location.

Comment (gergesh) [reply Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/ImageViewerController.java]: How do I get the context in the ImageViewerPresenter?

Comment (Adamantcheese) [reply Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/ImageViewerController.java]: You don't need the exact context for system level managers, so using `AndroidUtils.getAppContext()` should work fine.

Comment (gergesh) [reply Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/controller/ImageViewerController.java]: > You don't need the exact context for system level managers, so using `AndroidUtils.getAppContext()` should work fine.

Alright, done. Thanks for the tip!

**Comment by Adamantcheese** [comment]
Looks good to me, I'll merge it in a second once I finish up making all the checkboxes in the application colored more correctly to match the theme.

**Pull request merged**